diff -uNrp DxLibMake.old/DxCompileConfig.h DxLibMake.new/DxCompileConfig.h
--- DxLibMake.old/DxCompileConfig.h	2021-01-31 17:36:24.000000000 +0900
+++ DxLibMake.new/DxCompileConfig.h	2021-07-13 01:00:57.419421751 +0900
@@ -182,11 +182,13 @@
 
 #ifndef __APPLE__
 #ifndef __ANDROID__
+#ifndef __linux__
 	#define WINDOWS_DESKTOP_OS
+#endif // __linux__
 #endif // __ANDROID__
 #endif // __APPLE__
 
-#if defined( DX_GCC_COMPILE ) || defined( __ANDROID__ ) || defined( __APPLE__ ) || defined( BC2_COMPILER )
+#if defined( DX_GCC_COMPILE ) || defined( __linux__ ) || defined( __ANDROID__ ) || defined( __APPLE__ ) || defined( BC2_COMPILER )
 	#define USE_ULL
 #endif
 
@@ -209,7 +211,7 @@
 	#endif // DX_NON_MEDIA_FOUNDATION
 #endif // WINDOWS_DESKTOP_OS
 
-#if defined( __ANDROID__ ) || defined( __APPLE__ )
+#if defined( __linux__ ) || defined( __ANDROID__ ) || defined( __APPLE__ )
 //#define DX_NON_2DDRAW
 #define DX_NON_ACM
 #define DX_NON_DSHOW_MP3
@@ -385,7 +387,7 @@
 
 
 
-#if defined( _WIN64 ) || defined( __ANDROID__ ) || defined( __APPLE__ )
+#if defined( _WIN64 ) || defined( __linux__ ) || defined( __ANDROID__ ) || defined( __APPLE__ )
 	#ifndef DX_NON_INLINE_ASM
 		#define DX_NON_INLINE_ASM
 	#endif
