diff -uNrp DxLibMake.old/Linux/DxSoundLinux.h DxLibMake.new/Linux/DxSoundLinux.h
--- DxLibMake.old/Linux/DxSoundLinux.h	2021-10-04 22:15:14.814281076 +0900
+++ DxLibMake.new/Linux/DxSoundLinux.h	2021-10-04 22:15:05.850319771 +0900
@@ -17,8 +17,8 @@
 #include "../DxLib.h"
 #include "../DxThread.h"
 
-#include <OpenAL/al.h>
-#include <OpenAL/alc.h>
+#include <AL/al.h>
+#include <AL/alc.h>
 
 #ifndef DX_NON_NAMESPACE
 
