diff --git a/DxLibMake/iOS/DxSoundiOS.cpp b/DxLibMake/iOS/DxSoundiOS.cpp
index 263c23b..fdf8b4e 100644
--- a/DxLibMake/iOS/DxSoundiOS.cpp
+++ b/DxLibMake/iOS/DxSoundiOS.cpp
@@ -399,6 +399,10 @@ static int SoundReleaseInfo_Process( void )
 	// 破棄すべきサウンドが無い場合は何もせずに終了
 	if( ind == -1 )
 	{
+
+		// クリティカルセクションの解放
+		CriticalSection_Unlock( &SoundSysData.PF.SoundReleaseCriticalSection ) ;
+
 		return 0 ;
 	}
 
