diff --git a/DxLibMake/iOS/DxSoundiOS.cpp b/DxLibMake/iOS/DxSoundiOS.cpp
index 263c23b..9ed8edc 100644
--- a/DxLibMake/iOS/DxSoundiOS.cpp
+++ b/DxLibMake/iOS/DxSoundiOS.cpp
@@ -616,7 +616,7 @@ END :
 // サウンドバッファ再生処理用スレッド
 static void *ALBufferPlayThreadFunction( void *argc )
 {
-	while( SoundSysData.PF.ProcessSoundThreadEndRequest == FALSE )
+	while( SoundSysData.PF.ProcessALBufferThreadEndRequest == FALSE )
 	{
 		// クリティカルセクションの取得
 		CRITICALSECTION_LOCK( &HandleManageArray[ DX_HANDLETYPE_SOUND ].CriticalSection ) ;
@@ -657,7 +657,7 @@ static void *ALBufferPlayThreadFunction( void *argc )
 // ストリームサウンド処理用スレッド
 static void *StreamSoundThreadFunction( void *argc )
 {
-	while( SoundSysData.PF.ProcessALBufferThreadEndRequest == FALSE )
+	while( SoundSysData.PF.ProcessSoundThreadEndRequest == FALSE )
 	{
 		// クリティカルセクションの取得
 		CRITICALSECTION_LOCK( &HandleManageArray[ DX_HANDLETYPE_SOUND ].CriticalSection ) ;
